pkg/cache/client/context.go

```diff
@@ -37,9 +37,9 @@ func WithShardInContext(parent context.Context, shard shard.Name) context.Contex
 // ShardFromContext returns the value of the shard key on the ctx,
 // or an empty Name if there is no shard key.
 func ShardFromContext(ctx context.Context) shard.Name {
-	shard, ok := ctx.Value(shardContextKey).(shard.Name)
+	s, ok := ctx.Value(shardContextKey).(shard.Name)
```

Comment:
revert. Not worth the effort

Reply:
done, but what is about alias `cshard` or similar?